http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -931,28 +942,28 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift: Parallel](
         originalDetachTimestampOpt = originalDiskOpt.map(_.getLastDetachTimestamp)
 
         _ <- deleteApp
-        _ <- if (!errorAfterDelete)
-          dbApp.app.status match {
-            // If the message is resubmitted, and this step has already been run, we don't want to re-notify the app creator and update the deleted timestamp
-            case AppStatus.Deleted => F.unit
-            case _ =>
-              appQuery.markAsDeleted(msg.appId, ctx.now).transaction.void >> authProvider
-                .notifyResourceDeleted(dbApp.app.samResourceId, dbApp.app.auditInfo.creator, msg.project)
-                .void
-          }
-        else F.unit
+
         // we now use the detach timestamp recorded prior to helm uninstall so we can observe when galaxy actually 'detaches' the disk from google's perspective
         getDisk = googleDiskService.getDisk(msg.project,
                                             zone,
                                             gkeInterp.getGalaxyPostgresDiskName(dbApp.app.appResources.namespace.name))
-        diskDetachResult <- streamUntilDoneOrTimeout(
+        _ <- streamUntilDoneOrTimeout(
```

Comment:
I guess we only need to do this "wait for detach" logic if we're actually deleting the disk.. probably doesn't hurt to do it either way though.